src/components/common/ViewerSidebar.jsx

```diff
@@ -175,6 +175,139 @@ class ViewerSidebar extends Component {
             }
 
             return lassoSelections.map((lS, i) => {
+                let lassoDownloadSubLoomButton = () => {
```

Comment:
Please let's not continue doing this. That `<Button/>` should be a new component. And linting should tell you that `==` is forbidden? Use `===`

Reply:
I'm OK to leave it like this for now since I already refactored all this file in an existing branch (`cleanup/373-frontend-stage4-viewersidebar`) that's ready